osu.Game/Screens/Play/GameplayClockContainer.cs

```diff
@@ -296,5 +303,23 @@ public HardwareCorrectionOffsetClock(IClock source, bool processSource = true)
             {
             }
         }
+
+        # region Storyboard outro logic
+
+        public IBindable<bool> HasStoryboardEnded => hasStoryboardEnded;
+
+        public bool HasTimeLeftInStoryboard => GameplayClock.CurrentTime <= StoryboardEndTime;
```

Comment:
unnecessary? should just read from `HasStoryboardEnabled.Value`

Reply:
Is `DimmableStoryboard.ContentDisplayed` dynamically updated after the end of the storyboard is reached? I'm not sure what's the best way to check if the storyboard has ended.

Reply:
instead of `HasTimeLeftInStoryboard`, use `!HasStoryboardEnded.Value`, that's what I mean.